draft-ietf-quic-http.md

```diff
@@ -520,9 +622,13 @@ permitted (e.g., idempotent actions like GET, PUT, or DELETE).
 ### Malformed Requests and Responses {#malformed}
 
 A malformed request or response is one that is an otherwise valid sequence of
-frames but is invalid due to the presence of extraneous frames, prohibited
-header fields, the absence of mandatory header fields, or the inclusion of
-uppercase header field names.
+frames but is invalid due to:
+
+- the presence of prohibited header fields or pseudo-header fields
+- the absence of mandatory pseudo-header fields
+- invalid values for pseudo-header fields
```

Comment:
If this is intended to be exhaustive, then you need to include ordering of pseudo-header fields and header fields.

And maybe the inclusion of a non-Final status code after a final status code.

Reply:
> If this is intended to be exhaustive, then you need to include ordering of pseudo-header fields and header fields.

Good call.

> And maybe the inclusion of a non-Final status code after a final status code.

This would be one instance of an invalid sequence of messages.